halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/providers/ecs/EcsAccountValidator.java

```diff
@@ -0,0 +1,42 @@
+package com.netflix.spinnaker.halyard.config.validate.v1.providers.ecs;
+
+import com.netflix.spinnaker.halyard.config.model.v1.node.Account;
+import com.netflix.spinnaker.halyard.config.model.v1.node.Validator;
+import com.netflix.spinnaker.halyard.config.model.v1.providers.ecs.EcsAccount;
+import com.netflix.spinnaker.halyard.config.problem.v1.ConfigProblemSetBuilder;
+import com.netflix.spinnaker.halyard.config.services.v1.AccountService;
+import com.netflix.spinnaker.halyard.config.services.v1.ConfigService;
+import com.netflix.spinnaker.halyard.config.services.v1.ProviderService;
+import com.netflix.spinnaker.halyard.core.problem.v1.Problem.Severity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.List;
+import java.util.Optional;
+
+@Component
+public class EcsAccountValidator extends Validator<EcsAccount> {
+    @Autowired
+    ProviderService providerService;
+
+    @Autowired
+    AccountService accountService;
+
+    @Autowired
+    ConfigService configService;
+
+    @Override
+    public void validate(ConfigProblemSetBuilder p, EcsAccount n) {
+        p.addProblem(Severity.WARNING, "Very limited validation for the ECS provider has been implemented.");
```

Comment:
Could you rephrase this to "This only validates that a corresponding AWS account has been created for your ECS account."

I think that'll potentially avoid some questions down the road when people see a WARNING in their deploy logs.

Reply:
@lwander Sorry, I'm new at this. I've squashed my commits and force pushed which seems to be the correct approach based on the contributing page, but that makes the comments in this PR look odd since they no longer reflect the code. Should I instead be just adding new commits, and they can be squashed when merged into master ?

Anyway, added the docs and updated the warning message.

Reply:
No worries, that's the right approach. I (personally) prefer keeping all commits separate until we merge for clarity, maybe we can update the contributor guide to reflect that.